src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalAmount = days * 40;
+  let amount = 0;
+  const d7 = 50;
+  const d3 = 20;
+
+  if (days >= 7) {
+    amount = totalAmount - d7;
+  } else if (days >= 3) {
+    amount = totalAmount - d3;
+  } else {
```

Comment:
u can move it to main code, to avoid to many else if

Reply:
done